CupOfJoin.py

```diff
@@ -0,0 +1,9 @@
+def join(*lists, sep='-'):
+    l = []
```

Comment:
Change all the variable names - dont use single line variables where you can not use them